src/dfp/fill-static-advert-slots.ts

```diff
@@ -33,6 +36,18 @@ const decideAdditionalSizes = (adSlot: HTMLElement): SizeMapping => {
 			phablet: [adSizes.outstreamDesktop, adSizes.outstreamGoogleDesktop],
 			desktop: [adSizes.outstreamDesktop, adSizes.outstreamGoogleDesktop],
 		};
+	} else if (name === 'article-end' && isInUsa()) {
+		return {
+			mobile: [adSizes.fluid],
+		};
+	} else if (
```

Comment:
We don't need the `else`'s since we are returning when the condition is true

Reply:
Jake and I aren't following what you mean, could you please explain more?

Reply:
I've added a suggestion below to show what I mean